hack/functest.sh

```diff
@@ -21,7 +21,7 @@ fi
 # -r: run suites recursively
 # --keepGoing: don't stop on failing suite
 # -requireSuite: fail if tests are not executed because of missing suite
-ACK_GINKGO_DEPRECATIONS=1.16.4 ./bin/ginkgo $NO_COLOR -v -r --keepGoing -requireSuite ./test/e2e
+ACK_GINKGO_DEPRECATIONS=1.16.4 ./bin/ginkgo/v1.16.4/ginkgo $NO_COLOR -v -r --keepGoing -requireSuite ./test/e2e
```

Comment:
Can we pass the version as argument from the Makefile?
Or create a symlink from ./bin/ginkgo to the versioned binary after download, and use ./bin/ginkgo here?